src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -452,7 +453,7 @@ export default class AircraftController {
             altitude: spawnPatternModel.altitude,
             speed: spawnPatternModel.speed,
             heading: spawnPatternModel.heading,
-            position: spawnPatternModel.position,
+            position: spawnPatternModel.position.toPositionModel(),
```

Comment:
As you know, this was resolved, but required the helper functions to be in _individual_ files, to avoid the circular import that was the cause of those nasty `super() must be null or defined, not undefined` errors.